README.md

```diff
@@ -68,12 +68,11 @@ The easiest way to install Spyder is:
 
 ### On Windows:
 
-- Or using one of these scientific Python distributions:
+- Using one of our [executable installers](https://github.com/spyder-ide/spyder/releases)
```

Comment:
The only reasonable way to readd Windows installers is if we produce self-contained installers with py2exe, cx_freeze or a similar project.

But I don't have time for that, so someone else will have to help me with it ;-)